indico/modules/events/editing/schemas.py

```diff
@@ -244,3 +244,17 @@ def _validate_file_types(self, file_types):
         event_conditions = editing_settings.get(event, 'review_conditions')
         if any(condition_types == set(event_condition) for event_condition in event_conditions.values()):
             raise ValidationError(_('Conditions have to be unique'))
+
+
+class SideMenuItemSchema(mm.Schema):
+    name = fields.String(required=True)
+    title = fields.String(required=True)
+    url = fields.String(required=True)
+    weight = fields.Int()
```

Comment:
Only return stuff that's needed on the client. Assuming ordering is done server-side, we don't need that here.